.github/workflows/attach-static-libs.yaml

```
@@ -38,8 +38,10 @@ jobs:
             target: aarch64-unknown-linux-gnu
           - os: macos-latest
             target: aarch64-apple-darwin
+            pre-build-cmd: MACOSX_DEPLOYMENT_TARGET=$(sw_vers -productVersion | sed 's/\..*/\.0/')
```

It is necessary to write the environment variable to the [`GITHUB_ENV`](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-environment-variable) file for following steps to pick up the variable.

```suggestion
            pre-build-cmd: echo "MACOSX_DEPLOYMENT_TARGET=$(sw_vers -productVersion | sed 's/\..*/\.0/')" >> "$GITHUB_ENV"
```